.palantir/revapi.yml

```diff
@@ -877,6 +877,27 @@ acceptedBreaks:
     - code: "java.field.serialVersionUIDChanged"
       new: "field org.apache.iceberg.util.SerializableMap<K, V>.serialVersionUID"
       justification: "Serialization is not be used"
+    - code: "java.method.nowStatic"
+      old: "method org.apache.iceberg.BaseMetastoreTableOperations.CommitStatus org.apache.iceberg.BaseMetastoreTableOperations::checkCommitStatus(java.lang.String,\
+        \ org.apache.iceberg.TableMetadata)"
+      new: "method org.apache.iceberg.BaseMetastoreTableOperations.CommitStatus org.apache.iceberg.BaseMetastoreTableOperations::checkCommitStatus(java.lang.String,\
+        \ java.lang.String, java.util.Map<java.lang.String, java.lang.String>, java.util.function.Function<java.lang.String,\
+        \ java.lang.Boolean>)"
+      justification: "Increased visibility and refactor methods for view operation"
+    - code: "java.method.numberOfParametersChanged"
```

Comment:
I am not sure about this. Do we need a deprecation, and default method for this to be backward compatible?